description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
+
+Instances of practical sessions are represented by laboratories, seminars or workshops.
```

Comment:
```suggestion
Instances of practical sessions are represented by lab sessions, seminars or workshops.
```

I wouldn't use "laboratories", since that term is usually reserved for physical rooms.